test/helpers/cilium.go

```diff
@@ -328,7 +328,13 @@ func (s *SSHMeta) PolicyDelAll() *CmdRes {
 
 // PolicyDel deletes the policy with the given ID from Cilium.
 func (s *SSHMeta) PolicyDel(id string) *CmdRes {
-	return s.ExecCilium(fmt.Sprintf("policy delete %s", id))
+	res := s.ExecCilium(fmt.Sprintf(
```

Comment:
What is the motivation for this change to this function? What does ` s.ExecCilium(fmt.Sprintf("policy delete %s", id))` not provide?